org.srb2.SRB2Kart.yaml

```diff
@@ -0,0 +1,94 @@
+command: srb2kart
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.srb2kart
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: game-music-emu
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+      - /lib/*.so
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.xz
+        sha256: aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d
+
+  - name: srb2kart
+    buildsystem: simple
+    build-options:
+      arch:
+        arm:
+          env:
+            ARCH_MAKE_ARGS: 'LINUX=1 NONX86=1'
+        aarch64:
+          env:
+            ARCH_MAKE_ARGS: 'LINUX64=1'
+        x86_64:
+          env:
+            ARCH_MAKE_ARGS: 'LINUX64=1 X86_64=1'
+    build-commands:
+      - make -C src -j $FLATPAK_BUILDER_N_JOBS NOUPX=1 SDL=1 $ARCH_MAKE_ARGS
+      - install -D -m 755 -t $FLATPAK_DEST/bin bin/Linux*/Release/lsdl2srb2kart
+      - install -D -m 644 srb2.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -D -m 755 srb2kart.sh $FLATPAK_DEST/bin/srb2kart
+      - install -D -m 644 -t $FLATPAK_DEST/share/applications $FLATPAK_ID.desktop
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/STJr/Kart-Public
+        tag: v1.2
+        commit: f856f18233da492c9446f70a2856e2e178ddfd1a
+      - type: script
+        commands:
+          - ls ~/.srb2kart/srb2.srb || zenity --error --title "Missing game assets" --text "Game assets not found, please add them to your ~/.var/app/org.srb2.SRB2Kart/.srb2kart folder. See https://github.com/STJr/Kart-Public/releases for download instructions." --ok-label "Close" --width=400
+          - lsdl2srb2kart "$@"
+        dest-filename: srb2kart.sh
+      - type: file
+        path: org.srb2.SRB2Kart.desktop
+      - type: file
+        path: org.srb2.SRB2Kart.metainfo.xml
+
+  - name: srb2kart-assets
+    buildsystem: simple
+    build-commands:
+      - install -t $FLATPAK_DEST/bin apply_extra
+    sources:
+      - type: extra-data
+        filename: Installer.exe
+        url: https://github.com/STJr/Kart-Public/releases/download/v1.2/srb2kart-v12-Installer.exe
+        sha256: 57284de43a2889c0a60ce8fee77df22095fcd31beef6130cbeed1ead0bebbfec
+        size: 200732615
+      - type: extra-data
+        filename: Patch.zip
+        url: https://github.com/STJr/Kart-Public/releases/download/v1.2/srb2kart-v12-Patch.zip
+        sha256: 0ed388a17e822c608e5528572a4c8723dbf3677c239019d239ead2df4bc0fb9e
+        size: 32397494
+      - type: script
+        commands:
+          - 7z x Installer.exe "*.kart" "*.srb" "mdls.dat" "mdls/*" > /dev/null && rm Installer.exe
+          - unzip -o Patch.zip "*.kart" "*.srb" "mdls.dat" "mdls/*" > /dev/null && rm Patch.zip
```

Comment:
I'm confused about this (Fedora Copr did it too).  But upstream's [website](https://mb.srb2.org/showthread.php?t=43708) says


> If you don't have the game, then you want the Full Install. If you already have v1.1 or higher, then you might want to use the Patch to save on download size, but it is otherwise functionally identical to using the full installer on the same folder.

So my interpretation is only the Installer.exe is needed.

Reply:
You might right. Let's try with just the installer.